Makefile

```diff
@@ -220,9 +221,12 @@ cni-plugins: get-cni-sources .out-stamp
 		"amazon/amazon-ecs-build-cniplugins:make"
 	@echo "Built amazon-ecs-cni-plugins successfully."
 
-run-integ-tests: test-registry gremlin container-health-check-image
+run-integ-tests: test-registry gremlin container-health-check-image run-sudo-integ-tests
 	. ./scripts/shared_env && go test -race -tags integration -timeout=7m -v ./agent/engine/... ./agent/stats/... ./agent/app/...
 
+run-sudo-integ-tests:
+	. ./scripts/shared_env && sudo -E ${GO_EXECUTABLE} test -race -tags 'sudo integration' -timeout=5m -v ./agent/engine/... ./agent/stats/... ./agent/app/...
```

Comment:
I think it simplifies the build logic if you only use a 'sudo' built target that implies integration. You wouldn't need to add the !sudo to every additional integ test.

Reply:
My concern about using just `sudo` was there may be different types of tests in the future that would need to run as `sudo`. Hence kept it to two tags. What do you think?

Reply:
I had considered that as well, but if we are interacting with host/docker resources its basically an integ test by definition.